app/src/main/java/co/nano/nanowallet/MainActivity.java

```diff
@@ -47,26 +47,21 @@
 import io.realm.RealmResults;
 
 public class MainActivity extends AppCompatActivity implements WindowControl, ActivityWithComponent {
-    private FragmentUtility mFragmentUtility;
-    private Toolbar mToolbar;
-    private TextView mToolbarTitle;
     protected ActivityComponent mActivityComponent;
-    private FrameLayout mOverlay;
-
     @Inject
     Realm realm;
-
     @Inject
     AccountService accountService;
-
     @Inject
     NanoWallet nanoWallet;
-
     @Inject
     SharedPreferencesUtil sharedPreferencesUtil;
-
     @Inject
     AnalyticsService analyticsService;
+    private FragmentUtility mFragmentUtility;
+    private Toolbar mToolbar;
+    private TextView mToolbarTitle;
+    private FrameLayout mOverlay;
```

Comment:
I think the fields shouldn't be moved to keep consistent with the styling from the other classes.

Reply:
My bad. This was Android Studio's reformatting.